### PR TITLE
added list support

### PR DESCRIPTION
**Pull request by fnc12**

### CODE REVIEW

File: dev/serializing_util.h [hunk 1]
```diff
@@ -197,8 +197,9 @@ namespace sqlite_orm {
             auto& context = get<2>(tpl);
 
             constexpr std::array<const char*, 2> sep = {", ", ""};
-            for(size_t i = 0, first = true; i < args.size(); ++i) {
-                ss << sep[std::exchange(first, false)] << serialize(args[i], context);
+            bool first = true;
+            for(auto it = args.begin(); it != args.end(); ++it) {
```
Comment (trueqbit) [inline dev/serializing_util.h:201]: Why not use `for(const auto& arg : args)`?

File: tests/statement_serializer_tests/logical_operators.cpp [hunk 2]
```diff
@@ -75,8 +75,26 @@ TEST_CASE("statement_serializer logical operators") {
...
         }
         SECTION("dynamic in") {
-            auto inValue = in(&User::id, {1, 2, 3});
-            stringValue = internal::serialize(inValue, context);
+            SECTION("initializer list") {
+                auto inValue = in(&User::id, {1, 2, 3});
+                stringValue = internal::serialize(inValue, context);
+            }
+            SECTION("vector") {
+                std::vector<int> vector;
```
Comment (trueqbit) [inline tests/statement_serializer_tests/logical_operators.cpp:83]: Why not directly initialize `std::vector<int> vector{1, 2, 3}`?

File: tests/statement_serializer_tests/logical_operators.cpp [hunk 3]
```diff
@@ -75,8 +75,26 @@ TEST_CASE("statement_serializer logical operators") {
...
+                vector.push_back(1);
+                vector.push_back(2);
+                vector.push_back(3);
+                auto inValue = in(&User::id, vector);
+                stringValue = internal::serialize(inValue, context);
+            }
+            SECTION("list") {
+                std::list<int> list;
```
Comment (trueqbit) [inline tests/statement_serializer_tests/logical_operators.cpp:91]: Why not directly initialize `std::list<int> list{1, 2, 3}`?

**Review by trueqbit — APPROVED** [review]

File: dev/serializing_util.h (hunk 1 above)
Comment (fnc12) [reply dev/serializing_util.h:201]: cause I have weird habits left from old C++ standards. Just fixed. Thanks

File: tests/statement_serializer_tests/logical_operators.cpp (hunk 2 above)
Comment (fnc12) [reply tests/statement_serializer_tests/logical_operators.cpp:83]: fixed

File: tests/statement_serializer_tests/logical_operators.cpp (hunk 3 above)
Comment (fnc12) [reply tests/statement_serializer_tests/logical_operators.cpp:91]: fixed

**Pull request merged**